### PR TITLE
RHMAP-11075 add User-Agent header

### PR DESCRIPTION
**Pull request by pb82**
# Motivation

Some of the older commands in FHC where using the `request` library directly (and not through `fhreq`) and they were not sendind the `User-Agent` header when calling millicore. The `User-Agent` must be set to `FHC`, in order to skip the CSRF check.

ping @wei-lee @pmdarrow @laurafitzgerald @JameelB

### CODE REVIEW

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* The code changes conform to the contributing & code style guides for this component (if they exist)
* A component owner should review your change (we've tagged these above)
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.
Fhcap PR will be generated automatically.
To deploy your changes to a cluster place comment in this PR with pattern [x][y] where x=deploy and y=clustername you wish to deploy. 
You can also deploy your change manually using fhcap-cli

File: lib/cmd/fhc/target.js [hunk 1]
```diff
@@ -137,11 +137,12 @@ function undefinedTargCleanup(cb, tar) {
 }
 
 function pingTarget(targ, callback) {
-  request(targ + "/box/srv/1.1/act/sys/auth/logout", function(err, response, body){
+  fhreq.GET(targ, "/box/srv/1.1/act/sys/auth/logout", i18n._("error pinging target"), function(err, parsed, _, response) {
```
Comment (pmdarrow) [inline lib/cmd/fhc/target.js]: Is this related to the User-Agent change?

Comment (pb82) [inline lib/cmd/fhc/target.js]: Yes, the `fhreq` wrapper will automatically add the `User-Agent` header to the request. Thats why we should always use it in FHC commands.

**Comment by pmdarrow** [comment]
LGTM.

**Comment by pb82** [comment]
@wei-lee I checked fhc and could not find any other places where raw request was used in combination with a missing user-agent
